src/config/dynamic-theme-fixes.config

```diff
@@ -3902,6 +3902,38 @@ INVERT
 
 ================================
 
+jumia.com
+jumia.co
+jumia.ma
+jumia.ci
+jumia.dz
+jumia.ug
+jumia.sn
+zando.co.za
+
+INVERT
+svg.ic[role="img"]:not([aria-label="Zando logo"])
+.logo > a > img
+.inbox > a:not(:nth-of-type(1)) > svg
+img[src*="jumia-group-logo.png"]
+img[src*="jumia_logo_small_checkout.png"]
+img[src*="/Jumia-Pay"]
+li.logo:nth-child(1) > .-i-jumia-logo
+img[src*="empty-cart.png"]
+.-header > img:nth-child(1)
+.-gy5 > .-fs0 > .vent-link[title*="Rewards"]
+.-gy5 > .-fs0 > .vent-link[title*="Pay"]
+.-gy5 > .-fs0 > .vent-link[title*="Food"]
+.-gy5 > .-fs0 > .vent-link[title*="Party"]
+.-gy5 > .-fs0 > .vent-link[title*="Now"]
+nav.s-menu > a.m-itm:nth-child(2) > svg
+nav.s-menu > a.m-itm:nth-child(3) > svg
+nav.s-menu > a.m-itm:nth-child(4) > svg
+nav.s-menu > a.m-itm:nth-child(5) > svg
```

Comment:
Icons in account page (except 1st and 6th icon)
![Account icons signed in](https://user-images.githubusercontent.com/14346945/100476947-a21c1b00-30ef-11eb-95bf-95a74fdc84ae.jpg)
![Account icons signed in (2)](https://user-images.githubusercontent.com/14346945/100476953-a47e7500-30ef-11eb-9abd-5f4b89ccf6e5.jpg)